client/myapp/src/Components/Pages/myOrders.js

```diff
@@ -26,44 +27,54 @@ function MyOrdersComp(props) {
     if (member) {
       getMemberOrders();
     }
-  })
+  },[])
 
-  const getMemberOrders = () => {
-    let ordersId = member.orders;
 
-    ordersId.forEach(element => {
-      getAllOrders(element);
-    });
-  }
+  const getMemberOrders = async() =>{
+    let resp1 = await utils.getMember(member._id)
+    console.log("member :"+ resp1.data)
+    let memberId = member._id
+    let resp = await ordersUtils.getByMemberId({memberId});
```

Comment:
if you are using axios I would not using await since it is blocking the load of the page until you will get the list of orders back. Instead just return the axios.get method and handle the promise as we did before using .then()

Reply:
Done